src/framework/uicomponents/view/internal/popupviewclosecontroller.cpp

```diff
@@ -115,6 +131,12 @@ bool PopupViewCloseController::eventFilter(QObject* watched, QEvent* event)
         }
     }
 
+    if (QEvent::ApplicationStateChange == event->type() && !m_isDialog) {

```

Comment:
There is a processing of the application state change signal in the init method. see [line 37](https://github.com/musescore/MuseScore/blob/master/src/framework/uicomponents/view/internal/popupviewclosecontroller.cpp#L37)
Is this connect not working?

Reply:
> There is a processing of the application state change signal in the init method

It did not work at one point when I was trying to reproduce the issue, however I have just checked and the issue does not exist on `master`. Not sure what I managed to do to reproduce the issue.

Apologies for this unneeded PR.

What is wished to be done regarding the following (from **Additional Notes**)?

> Is `doUpdateEventFiletrs()` meant to be `doUpdateEventFilters()`? If so I would be happy to change this in ~either this PR or~ a new PR.

Reply:
> What is wished to be done regarding the following (from Additional Notes)?

@iwoithe yes, should be doUpdateEventFilters

> Apologies for this unneeded PR.

Don't worry. Thanks for your help